src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_dashboards.tsx

```diff
@@ -138,19 +141,31 @@ export const RuleDashboards = ({ contentManagement }: Props) => {
     label: dashboard.attributes.title,
   });
 
+  useEffect(() => {
```

Comment:
How about using the observable variant of the `getActiveSpace` method to avoid the `useEffect`?

```tsx
const activeSpace = useObservable(spaces.getActiveSpace$().pipe(map(({ id }) => id)), '*');
```

The observable chain could be modified to account for the `services` prop being `undefined`, but it's probably worth making it required seeing the usages

Reply:
I think this should be handled automatically by the backend, and not have the UI have knowledge about Spaces. Does this supported by `contentManagementService.client.search` if you omit the `spaces` option?

Reply:
Done! @umbopepato @cnasikas please check.